django/contrib/gis/db/models/fields.py

```diff
@@ -312,6 +312,12 @@ def get_distance(self, value, lookup_type, connection):
         """
         return connection.ops.get_distance(self, value, lookup_type)
 
+    def get_db_prep_value(self, value, connection, *args, **kwargs):
+        return connection.ops.Adapter(
+            super(GeometryField, self).get_db_prep_value(value, connection, *args, **kwargs),
+            **({'geography': True} if self.geography else {})
```

Comment:
Is there a test covering this line? I commented it out but didn't see any failures in a quick check.